pulsar-client-go/pulsar/reader_test.go

```diff
@@ -218,3 +219,49 @@ func TestReaderCompaction(t *testing.T) {
 	assert.Nil(t, msg)
 	assert.NotNil(t, err)
 }
+
+func TestReaderHasNext(t *testing.T) {
+	topic := fmt.Sprintf("TestReaderHasNext-%d", rand.Int())
+	ctx := context.Background()
+
+	client, err := NewClient(ClientOptions{
+		URL:                     "pulsar://localhost:6650",
+		//Logger: func(level logutil.LoggerLevel, file string, line int, message string) {
```

Comment:
Do you want to checkin with these comments

Reply:
Good point.. :) removing these